src/Symfony/Component/Yaml/Tests/DumperTest.php

```diff
@@ -910,6 +910,11 @@ public function testDumpNullAsTilde()
         $this->assertSame('{ foo: ~ }', $this->dumper->dump(['foo' => null], 0, 0, Yaml::DUMP_NULL_AS_TILDE));
     }
 
+    public function testCanForceQuotesOnValues()
+    {
+        $this->assertSame('{ foo: "bat", bar: 23 }', $this->dumper->dump(['foo' => 'bat', 'bar' => 23], 0, 0, Yaml::DUMP_FORCE_DOUBLE_QUOTES_ON_VALUES));
```

Comment:
It would be interesting to have a test showing how it behaves with boolean values, but mostly a string already containing doubles quotes

Reply:
Done BTW.

Reply:
Perfect, thanks!